src/sass/_theme_rst.sass

```
@@ -481,7 +481,16 @@
     font-weight: 700
     border-radius: $base-line-height / 6
     padding: $base-line-height / 10 $base-line-height / 4
-    margin: auto $base-line-height / 12
+    margin: auto 0
+  kbd, .kbd
```

```suggestion
  :not(kbd) > kbd,
  :not(kbd) > .kbd,
  :not(.kbd) > kbd,
  :not(.kbd) > .kbd
```
To prevent the case mentioned in https://github.com/readthedocs/sphinx_rtd_theme/pull/677#issuecomment-658637531

This can be simplified if we only apply styling to `kbd` element or `.kbd` classname. I'm not sure we need both.